test/box/error.result

```diff
@@ -1060,3 +1060,38 @@ copy1:set_prev(e1)
 copy1:set_prev(e1)
  | ---
  | ...
+-- net.box returns a copy of the internal error
+box.cfg{listen = 3301}
```

Comment:
Again, the test should be written in luatest. Since this one requires `box.cfg`, it should reside in `test/box-luatest`.

Reply:
Done. See test/box-luatest/gh_6632_result_returns_a_copy_of_the_internal_error_test.lua.